website/docs/developer/research/research-orca-delete-functionality.md

```diff
@@ -3,47 +3,67 @@ id: research-orca-delete-functionality
 
     
     This can be implemented in the existing lambda function that will be used to delete the data from ORCA.
+    Before deleting from CMR and Cumulus, The UI should show a warning message to the user for additional confirmation.
 
 ### API gateway
 
-API gateway is currently used by end users for creating reconciliation reports and getting status update of recovery granules and jobs. This can be used for creating API calls made by end users for deletion. The API gateway should act as a trigger to the lambda functions that actually perform the deletion in the backend. An API gateway with two different resources need to be created- one for the soft delete API call and one for the hard delete API call.
-Check API gateway [examples](https://github.com/nasa/cumulus-orca/tree/develop/modules/api-gateway) currently used by ORCA. 
+API gateway is currently used by end users for creating reconciliation reports and getting status update of recovery granules and jobs. This can be used for creating API calls made by end users for deletion. The API gateway should act as a trigger to the lambda functions that actually perform the deletion in the backend. An API gateway with two different resources need to be created- one for the soft delete API call and one for the hard delete API call. In order to deploy this via terraform, a private rest api with two resources need to be created. In case of using private API, the vpc endpoint ID for the gateway will be required. The REST API should use a `POST` http method and the integration type should be `AWS`. If lambda function is used with the API, a lambda permission resource should be created in terraform giving access to `lambda:InvokeFunction` action so that the API can trigger the lambda. In addition, the api gateway resource policy can be used to restrict certain IP addresses and VPC if needed from accessing the API endpoint. Check API gateway [examples](https://github.com/nasa/cumulus-orca/tree/develop/modules/api-gateway) currently used by ORCA. 
 
 ### Lambda function
 
 The delete functionality can be performed by two lambda functions based on initial research. One of the lambda functions will be used to perform soft delete- delete from S3 bucket but only flag the granule in the database for deletion after 90 days. The lambda should also verify with CMR and/or Cumulus that the granule has been deleted from those locations prior to allowing the delete to proceed in ORCA. The lambda will be triggered by an API call made by the user once the user is authenticated with Launchpad.
 
 The other lambda function can be used to perform a physical delete- delete from S3 glacier as well as from the database. Similar to soft delete, it should first verify with CMR and/or Cumulus that the granule has been deleted from those locations before deleting from ORCA. This will require further discussion with Cumulus team before performing the actual work.
```

Comment:
I would change `require further discussion with Cumulus team` to include ORCA team and ORCA Working group discussions

Reply:
changed